tests/backends/database/test_mongo.py

```diff
@@ -0,0 +1,206 @@
+"""Tests for Ralph mongo database backend"""
+
+import pytest
+from bson.objectid import ObjectId
+from pymongo import MongoClient
+from pymongo.errors import BulkWriteError
+
+from ralph.backends.database.mongo import MongoDatabase
+from ralph.exceptions import BadFormatException
+
+from tests.fixtures.backends import (
+    MONGO_TEST_COLLECTION,
+    MONGO_TEST_DATABASE,
+    MONGO_TEST_URI,
+)
+
+
+def test_backends_database_mongo_database_instantiation():
+    """Test the Mongo backend instantiation."""
+
+    assert MongoDatabase.name == "mongo"
+
+    backend = MongoDatabase(
+        connection_uri=MONGO_TEST_URI,
+        database=MONGO_TEST_DATABASE,
+        collection=MONGO_TEST_COLLECTION,
+    )
+
+    assert isinstance(backend.client, MongoClient)
+    assert hasattr(backend.client, MONGO_TEST_DATABASE)
+    database = getattr(backend.client, MONGO_TEST_DATABASE)
+    assert hasattr(database, MONGO_TEST_COLLECTION)
+
+
+def test_backends_database_mongo_get_method(mongo):
+    """Test the mongo backend get method."""
+
+    # Create records
+    documents = MongoDatabase.to_documents([{"id": "foo"}, {"id": "bar"}])
```

Comment:
What do you think of using the `fixed_dictionaries` strategy from hypothesis to generate statements?

Reply:
Maybe it's too convoluted for those tests? Also it might be less efficient.

Reply:
you're right, let's keep the KISS principle in mind